superset/charts/schemas.py

```diff
@@ -1159,10 +1151,7 @@ class Meta:  # pylint: disable=too-few-public-methods
     )
     filters = fields.List(fields.Nested(ChartDataFilterSchema), allow_none=True)
     granularity = fields.String(
-        metadata={
-            "description": "Name of temporal column used for time filtering. "
-            "For legacy Druid datasources this defines the time grain."
-        },
+        metadata={"description": "Name of temporal column used for time filtering. "},
```

Comment:
@michael-s-molina I _believe_ your question is somewhat akin to the conversation with @villebro where the term "granularity" is overloaded. Per the augmented `description` you can see it referred to the time gran for Druid NoSQL connector (which is now obsolete), however it also is used to represent the temporal time column for the SQL connector.